src/corona_hakab_model/sparse_base.py

```diff
@@ -48,10 +48,12 @@ def __getitem__(self, item: Tuple[int, int]) -> Optional[Tuple[float, float]]:
         """
 
     @abstractmethod
-    def manifest(self, sample: np.ndarray = None) -> ManifestBase:
+    def manifest(self, sample: np.ndarray = None, global_prob_factor: float = 1) -> ManifestBase:
         """
         Create a manifested sparse matrix from sample rolls.
         if sample is None, a new rolls array is created
+        global_prob_factor denotes a global probability coefficient to apply to all probs. 0.5 means that each cell has
+        only half as much chance to manifest. A global_prob_factor of 0 is an error.
```

Comment:
we do want to allow global prob factor of 0.
should check for it, and if so, switch sample to np.zeroes

Reply:
a. the sample should change to np.ones
b. I'd rather users check on their own because calling manifest with ones just adds overhead that could be avoided

Reply:
I want to discourage calling this function with a factor zero because that can usually be optimized elsewhere